student-info-system/sis/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from phone_field import PhoneField
+
+
+class Semester(models.Model):
+    name = models.CharField('Semester Name', max_length=20, default='xxx')
+    date_registration_opens = models.DateField('Registration Opens')
+    date_started = models.DateField('Classes Start')
+    date_last_drop = models.DateField('Last Drop')
+    date_ended = models.DateField('Classes End')
+
+    def __str__(self):
+        return self.name
+
+
+class Person(models.Model):
+    name = models.CharField('Name', max_length=128, help_text='Your Full Name')
+    address = models.CharField('Address',
+                               max_length=256,
+                               help_text='Your Mailing Address',
+                               blank=True)
+    email = models.EmailField(help_text='Your Primary Email Address',
+                              unique=True)
+    phone = PhoneField(help_text='Your Primary Contact Phone', blank=True)
+    is_active = models.BooleanField('Is Active?', default=False)
```

Comment:
Why so many? Are we customizing who can do these things rather than having explicit admin, student, and professor roles?

Reply:
Oh also we be interested in using the available meta options to set permissions, if you havent checked it out yet heres a [link](https://docs.djangoproject.com/en/3.1/ref/models/options/)